app/graph/mutations/tasks_order_mutation.rb

```diff
@@ -0,0 +1,14 @@
+TasksOrderMutation = GraphQL::Relay::Mutation.define do
+  name 'TasksOrder'
+
+  input_field :tasks, !JsonStringType
+
+  return_field :success, types.Boolean
+
+  return_field :errors, JsonStringType
+
+  resolve -> (_root, inputs, _ctx) {
+    errors = Task.order_tasks(inputs[:tasks])
+    { success: true, errors: errors }
```

Comment:
`success: !errors`

Reply:
I did a bulk sort so may be one of the ids have an issue like record not found, user has no ability to update, etc ... so I catch any errors while saving the item in errors key